src/services/tracing.ts

```diff
@@ -346,7 +348,9 @@ const resolveFunctionSpanOptions = ({
     for (const key in params) {
       // @ts-ignore-next-line no-implicit-any error
       const value = params[key]
-      attributes[`${SemanticAttributes.CODE_FUNCTION}.params.${key}`] = value
+      attributes[`${SemanticAttributes.CODE_FUNCTION}.params.${key}`] = redactFnArgs
```

Comment:
I dont think this should be the solution... there is a config for redacted params.. also this is not flexible enough to have the non-sensitive params